tests/system/general/test_35_MaxwellCircuit.py

```diff
@@ -115,3 +115,26 @@ def test_08_import_netlist(self):
         self.aedtapp.insert_design("SchematicImport")
         self.aedtapp.modeler.schematic.limits_mils = 5000
         assert self.aedtapp.create_schematic_from_netlist(self.netlist_file1)
+
+    def test_09_create_voltage_source(self):
+        name_type = ["voltage_source", 123, 1.23]
+        for item in name_type:
+            self.component = self.aedtapp.modeler.schematic.create_v_sin(name=item, value=3.14, angle=90)
```

Comment:
```suggestion
            component = self.aedtapp.modeler.schematic.create_v_sin(name=item, value=3.14, angle=90)
```
@DaveTwyman why self.component? it's not a property of the test class. Please remove all the other occurances.

Reply:
@gmalinve , Good point, with hindsight there was a lot wrong with test_35.